src/go/cmd/token-vendor/tokensource/gcp.go

```diff
@@ -44,15 +44,19 @@ func NewGCPTokenSource(ctx context.Context, client *http.Client, project, sa str
 // Token returns an access token for the configured service account and scopes.
 //
 // API: https://cloud.google.com/iam/docs/reference/credentials/rest/v1/projects.serviceAccounts/generateAccessToken
-func (g *GCPTokenSource) Token(ctx context.Context) (*TokenResponse, error) {
+func (g *GCPTokenSource) Token(ctx context.Context, saName string) (*TokenResponse, error) {
 	req := iam.GenerateAccessTokenRequest{Scope: g.scopes}
+	resource := g.resource
```

Comment:
can we rename resource to defaultResource or similar?

Reply:
And match the name with the parameter name

Reply:
Done.